README.md

```diff
@@ -118,118 +116,104 @@ The plugin will write out all metrics serialized as JSON to the specified file.
 See full task manifests using the mock and psutil plugins which are available in [examples/tasks](examples/tasks)  
 
 
-Example of running mock collector plugin, passthru processor plugin, and writing data as a JSON to file:
+Example of running [psutil collector plugin](https://github.com/intelsdi-x/snap-plugin-collector-psutil), [movingaverage processor plugin](https://github.com/intelsdi-x/snap-plugin-processor-movingaverage), and writing data as a JSON to file:
 
-Make sure that your `$SNAP_PATH` is set, if not:
+Set up the [Snap framework](https://github.com/intelsdi-x/snap/blob/master/README.md#getting-started)
+
+Make sure that [snapd](https://github.com/intelsdi-x/snap/blob/master/docs/SNAPD.md) and [snapctl](https://github.com/intelsdi-x/snap/blob/master/docs/SNAPCTL.md) are installed.
+
+In one terminal window, run the Snap daemon (in this case with logging set to 1 and trust disabled):
 ```
-$ export SNAP_PATH=<snapDirectoryPath>/build
+$ snapd -l 1 -t 0
```

Comment:
We should start using service commands whenever possible:
https://github.com/intelsdi-x/snap-plugin-collector-disk/blame/master/README.md#L95-L98